stdlib/3/copyreg.pyi

```diff
@@ -0,0 +1,9 @@
+
+from typing import TypeVar, Callable, Union, Tuple, Any, Optional
```

Comment:
This is missing a copy of things that exist in at least 3.8:
```
__all__ = ["pickle", "constructor",
           "add_extension", "remove_extension", "clear_extension_cache"]
```
You should either add them or add a `__getattr__` function to the stub to indicate it is incomplete.

Reply:
They exist back in 2.7, the whole file has received very few changes since then. As they're not included in the docs for 2/3, should I mark them `# undocumented`?

Reply:
No need for `# undocumented`, because objects in `__all__` should always be included: https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#what-to-include